Mage.Sets/src/mage/cards/e/EvolvedSleeper.java

```diff
@@ -0,0 +1,126 @@
+ */
+public final class EvolvedSleeper extends CardImpl {
+
+    public EvolvedSleeper(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{B}");
+
+        this.subtype.add(SubType.HUMAN);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(1);
+
+        // {B}: Evolved Sleeper becomes a Human Cleric with base power and toughness 2/2.
+        Ability ability = new SimpleActivatedAbility(new AddCardSubTypeSourceEffect(
+                Duration.Custom, SubType.HUMAN, SubType.CLERIC
+        ).setText("{this} becomes a Human Cleric"), new ManaCostsImpl<>("{B}"));
+        ability.addEffect(new SetPowerToughnessSourceEffect(
+                2, 2, Duration.Custom, SubLayer.SetPT_7b
+        ).setText("with base power and toughness 2/2"));
+        this.addAbility(ability);
+
+        // {1}{B}: If Evolved Sleeper is a Cleric, put a deathtouch counter on it and it becomes a Phyrexian Human Cleric with base power and toughness 3/3.
+        this.addAbility(new SimpleActivatedAbility(
+                new EvolvedSleeperClericEffect(), new ManaCostsImpl<>("{1}{B}")
+        ));
+
+        // {1}{B}{B}: If Evolved Sleeper is a Phyrexian, put a +1/+1 counter on it, then you draw a card and you lose 1 life.
+        this.addAbility(new SimpleActivatedAbility(
+                new EvolvedSleeperPhyrexianEffect(), new ManaCostsImpl<>("{1}{B}{B}")
+        ));
+    }
+
+    private EvolvedSleeper(final EvolvedSleeper card) {
+        super(card);
+    }
+
+    @Override
+    public EvolvedSleeper copy() {
+        return new EvolvedSleeper(this);
+    }
+}
+
+class EvolvedSleeperClericEffect extends OneShotEffect {
+
+    EvolvedSleeperClericEffect() {
+        super(Outcome.Benefit);
+        staticText = "if {this} is a Cleric, put a deathtouch counter on it " +
+                "and it becomes a Phyrexian Human Cleric with base power and toughness 3/3";
+    }
+
+    private EvolvedSleeperClericEffect(final EvolvedSleeperClericEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public EvolvedSleeperClericEffect copy() {
+        return new EvolvedSleeperClericEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Permanent permanent = source.getSourcePermanentIfItStillExists(game);
+        if (permanent == null || !permanent.hasSubtype(SubType.CLERIC, game)) {
+            return false;
+        }
+        permanent.addCounters(CounterType.DEATHTOUCH.createInstance(), source.getControllerId(), source, game);
+        game.addEffect(new AddCardSubTypeSourceEffect(
+                Duration.Custom, SubType.PHYREXIAN, SubType.HUMAN, SubType.CLERIC
+        ), source);
+        game.addEffect(new SetPowerToughnessSourceEffect(
+                3, 3, Duration.Custom, SubLayer.SetPT_7b
+        ), source);
+        return true;
+    }
+}
+
+class EvolvedSleeperPhyrexianEffect extends OneShotEffect {
```

Comment:
Is this supposed to happen as one effect since the card text has a comma between the +1/+1 gain and the draw? Or should it be split over two so that anything that cares about the draw or the life-loss sees the evolved sleeper as already having the +1/+1?

Reply:
why would it matter whether it has the counter?

Reply:
I fixed one or two issues like this. See the checked off one https://github.com/magefree/mage/issues/6761. There was another but can't find it atm. The second part of the effect (draw and lose life) may interact differently with other cards if there is a creature with a +1/+1 counter on the board. Having them as one effect will cause the +1/+1 gain and the draw+lose life to occur simultaneously rather than sequentially.

Reply:
oh right, any objection to adding `game.applyEffects()` in between?

Reply:
> oh right, any objection to adding `game.applyEffects()` in between?

I can't comment one way or another. For the linked issue I split the effect across two. Not sure of the implication of `game.applyEffects()`.

Reply:
the only way I can see this mattering is if the creature's P/T with the counter is relevant, which is what `game.applyEffects()` will do, as the creature should show as having a counter by the time the draw happens anyway

Reply:
I think it should be added to be technically correct.

Even if there isn't a card right now that cares about it, there might be one in the future.

Reply:
I don't disagree, I'll add it in

Reply:
Can you please add the apply.Effect()

Reply:
I did

Reply:
My bad. I missed it since this didn't my show as being outdated or resolved.